### PR TITLE
Release v1.0

### PR DESCRIPTION
**Pull request by MahdiAghaali**
I and my partner @Alhajideen Created an application using ruby for our capstone module

In this project, we create a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but you will also prepare a database with the structure of a table analogical to our program's class structure.

for more information about the project and features, you can read the [Readme.md](https://github.com/MahdiAghaali/ruby-capstone-catalog/blob/dev/README.md) file

[Live Demonstration(also in readme)](https://youtu.be/2EueHnC4bTY)

### CODE REVIEW

File: Modules/author.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    @id = SecureRandom.uuid
+    @first_name = first_name
+    @last_name = last_name
+    @items = items
+  end
+
+  def add_item(item)
+    @items << item
+    item.genre = self
+  end
```
Comment (mikethreels) [inline Modules/author.rb:18]: - [ ] this piece of code seems to be copied from your genre.rb file as the line `item.genre = self` should be `item.author = self`

File: Modules/item.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+require 'date'
+require 'securerandom'
+
+class Item
+  attr_accessor :genre, :author, :source, :label, :publish_date, :id, :archived
```
Comment (mikethreels) [inline Modules/item.rb:5]: - [ ] the genre, author and label should be added to the attr_reader instead of the accessor.

File: Modules/item.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  def move_to_archive
+    @archived = true if can_be_archived?
+    nil
+  end
+
+  def genres=(genre)
+    @genre = genre
+    genre.items.push(self) unless genre.items.include?(self)
+  end
```
Comment (mikethreels) [inline Modules/item.rb:34]: - [ ] you set up the 1-to-many relationship for the genre but the same should be done for the author and label

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @MahdiAghaali, 

In my opinion, your project is ready for presentation! ⚡🔥 There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**